.github/workflows/publish-and-sync-repos.yml

```diff
@@ -1,11 +1,12 @@
-name: Publish
+name: Publish and Sync Repos
 # Based on: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
 on:
   push:
-    tags:
```

Comment:
Let's keep this as is?

I think the goal is that we publish and sync repos at the same time on the same cadence.